test/conftest.py

```diff
@@ -1,32 +1,47 @@
 import json
 import pytest
 
+# import os
+
+from elasticsearch import Elasticsearch
 from pymongo import MongoClient
 from pymongo.errors import ServerSelectionTimeoutError
 
 from fhirstore import FHIRStore
 
 DB_NAME = "fhirstore_test"
+# MONGO_USERNAME = os.getenv("MONGO_USERNAME")
+# CLIENT_PASSWORD = os.getenv("CLIENT_PASSWORD")
 
 
 @pytest.fixture(scope="session")
 def store():
-    client = MongoClient(serverSelectionTimeoutMS=5)
+    client = MongoClient(username="arkhn", password="SuperSecurePassword2019")
```

Comment:
use environment variables instead (uncomment the lines above)

Reply:
Environment variables are now used.